ts/Core/Axis/Axis.ts

```diff
@@ -4121,7 +4120,11 @@ class Axis implements AxisComposition, AxisLike {
         axis.plotLinesAndBandsGroups = {};
 
         // Shorthand types
-        axis.positiveValuesOnly = !!(axis.logarithmic && !options.allowNegativeLog);
+        axis.positiveValuesOnly = !!(
```

Comment:
Please revert this and the other related changes, see https://github.com/highcharts/highcharts/issues/13914#issuecomment-671298850 . @bre1470 .